addons/l10n_pe_website_sale/views/templates.xml

```diff
@@ -0,0 +1,107 @@
+                    title="Changing Identification type is not allowed once document(s) have been issued for your account. Please contact us directly for this operation."/>
+                <input name="l10n_latam_identification_type_id"
+                    class="form-control"
+                    t-att-value="partner.l10n_latam_identification_type_id.id"
+                    type='hidden'/>
+            </t>
+        </div>
+
+    </template>
+
+    <template id="partner_address_info" name="Peruvian partner address">
+
+        <!-- show city -->
+        <div t-attf-class="mb-3 #{error.get('city_id') and 'o_has_error' or ''} col-lg-6 div_city_id"
+            t-attf-style="#{(country and country.code != 'PE') and 'display: none' or ''}">
+            <label class="col-form-label" for="city_id">City</label>
+            <select id="city_id"
+                name="city_id"
+                t-attf-class="form-select #{error.get('city_id') and 'is-invalid' or ''}"
+                data-init="1">
+                <t t-foreach="state_cities" t-as="city">
+                    <option t-att-value="city.id"
+                        t-att-selected="city.id == ('city_id' in checkout and checkout['city_id'] != '' and int(checkout['city_id']))">
+                        <t t-out="city.name" />
+                    </option>
+                </t>
+            </select>
+        </div>
+
+        <!-- show district -->
+        <div t-attf-class="mb-3 #{error.get('l10n_pe_district') and 'o_has_error' or ''} col-lg-6 div_district"
+            t-att-style="not city and 'display: none'">
+            <label class="col-form-label" for="l10n_pe_district">District</label>
+            <select id="l10n_pe_district"
+                name="l10n_pe_district"
+                t-attf-class="form-select #{error.get('l10n_pe_district') and 'is-invalid' or ''}"
+                data-init="1">
+                <t t-foreach="city_districts" t-as="district">
+                    <option t-att-value="district.id"
+                        t-att-selected="district.id == ('l10n_pe_district' in checkout and checkout['l10n_pe_district'] != '' and int(checkout['l10n_pe_district']))">
+                        <t t-out="district.name" />
+                    </option>
+                </t>
+            </select>
+        </div>
+
+    </template>
+
+    <template id="address" inherit_id="website_sale.address">
+        <xpath expr="//input[@name='vat']/.." position="before">
+            <t t-if="mode[1] == 'billing'" position="inside">
+                <t t-if="res_company.country_id.code == 'PE'">
+                    <t t-set="partner" t-value="website_sale_order.partner_id" />
+                    <t t-call="l10n_pe_website_sale.partner_info" />
+                </t>
+            </t>
+        </xpath>
+        <label for="vat" position="attributes">
+            <attribute name="t-if">res_company.country_id.code != 'PE'</attribute>
+        </label>
```

Comment:
This seems like the kind of change where multiple templates override may conflict (and the last one will win).
```suggestion
        <label for="vat" position="replace">
            <t t-if="res_company.country_id.code != 'PE'">
                $0
            </t>
        </label>
```

Reply:
If I perform this change I get the following error, this also shows up during a clean installation not just while updating:

`odoo.tools.convert.ParseError: while parsing None:75`
`Error while parsing or validating view:`
`Element '<label for="vat">' cannot be located in parent view`

Reply:
```suggestion
        <label for="vat" position="replace">
            <t t-if="res_company.country_id.code != 'PE'">
                $0
            </t>
            <t t-else="">
                <label class="col-form-label label-optional" for="vat">
                    Number
                </label>
            </t>
        </label>
```
It's because of the next xpath node.  Does it work if you integrate both of them like that ?

Reply:
It does work as long as the website has a Peruvian company but if I were to change the website domain to the default website it behaves strangely and instead of the vat options it displays a "$0". I believe it's best to leave it as it originally was.

Reply:
Strange, i guess we'll investigate that and improve the hooks in a future version.
Note that your xpath will conflict with the one defined in `l10n_ar_website_sale` (but I suppose there isn't many database that will have the two localisations together, if that's even supported)

Reply:
@Feyensv We have the same thing for PE, EC and AR already.  (CL probably not because we use the extra page)  Would not there be a way to provide a non-stored computed field on the partner website_vat_required_something?  We will need to solve this problem anyways.

Reply:
Why would there be a need of a computed field for that ?
It should be a simple method on the ´res.country´ model, called once in the base template.

Reply:
@AndreaGeraldo I have similar task and tried this too, and it seems that it displays "$0" because the `<t>` perceives them as a string when it's broken into multiple lines. In my case it works if I group them in one line, like this:
`<t t-if="res_company.country_id.code != 'PE'">$0</t>`

Reply:
@luistorresm Could you try this?

Reply:
I did it for EC since @yosuanicolaus told me about it.
https://runbot.odoo.com/runbot/batch/1253694
I tried it with AR and EC installed on the runbot and it seems to work